darwin/datatypes.py

```diff
@@ -284,6 +284,53 @@ def full_path(self) -> str:
         return construct_full_path(self.remote_path, self.filename)
 
 
+@dataclass(frozen=True, eq=True)
+class ConversionError:
+    """
+    Represents a failed conversion to YOLO format.
+
+    Attributes
+    ----------
+    reason : str
+        The reason on why the conversion failed.
+    annotation : Union[Annotation, VideoAnnotation]
+        The annotation that failed to be converted.
+    filename : Path
+        The path of the ``AnnotationFile`` holding the failed ``Annotation``.
+    """
+
+    reason: str
+    annotation: Union[Annotation, VideoAnnotation]
+    filename: Path
+
+
+@dataclass(frozen=True, eq=True)
+class YoloAnnotation:
```

Comment:
This is a "type" that is only useful for the YOLO exporting logic, and as such I would keep it there.

Reply:
done